lib/search_cop_grammar.rb

```diff
@@ -112,8 +116,11 @@ def evaluate
 
   class AndOrExpression < BaseNode
     def evaluate
-      [elements.first.evaluate, elements.last.evaluate]
-        .inject(query_info.scope.reflection.default_operator)
+      default_operator = :and
+      if query_options.member?(:default_operator) && [:and, :or].include?(query_options[:default_operator])
```

Comment:
we at least shouldn't fail silently when the default operator is `"AND"`, `"OR"`, `:AND`, etc.

Reply:
I was wondering how should handle exceptions here, should I raise an UnknownAttribute exception?

Reply:
simply add an exception (UnknownDefaultOperator) class in lib/search_cop.rb.